helm_deploy/hmpps-tier/templates/recalcuate-all-cases.yaml

```
@@ -17,7 +17,7 @@ spec:
               args:
                 - /bin/sh
                 - -c
-                - 'AUTH_TOKEN=$(curl --location --request POST "$AUTH_BASE_URL/oauth/token?grant_type=client_credentials" --header "Authorization: Basic $(echo -n $CLIENT_ID:$CLIENT_SECRET | base64)"  | jq -r .access_token)'
+                - 'AUTH_TOKEN=$(curl --location --request POST "$AUTH_BASE_URL/oauth/token?grant_type=client_credentials" --header "Authorization: Basic $(echo -n $CLIENT_ID:$CLIENT_SECRET | base64 -w 0 )"  | jq -r .access_token)'
```

Might be easier to use --user, and let curl do it for you:
```suggestion
                - 'AUTH_TOKEN=$(curl --location --request POST "$AUTH_BASE_URL/oauth/token?grant_type=client_credentials" --user "$CLIENT_ID:$CLIENT_SECRET"  | jq -r .access_token)'
```